tools/topology/topology2/include/pipelines/cavs/mixout-gain-ctc-dai-copier-playback.conf

```diff
@@ -0,0 +1,125 @@
+#
```

Comment:
@eddy1021 I have just introduced a new feature in the topology2 compiler to make it easier to extend extisting pipelines with one or more processing modules without having to copy & paste everything. For example, in your case you are adding the ctc module to the mixout-gain-dai-copier-playback.conf pipeline. Using the SubTreeCopy feature would make it easier for reviewers to see that you are simply extending the pipelines with an additional CTC module.

I have an example usage of this SubTreeCopy feature in my PR: https://github.com/thesofproject/sof/pull/9354. Please let me know if you have any questions.

Reply:
Hi @ranj063 , I tried to use the SubTreeCopy feature([change](https://github.com/thesofproject/sof/pull/8840/commits/645b27acf7726b83f7865ade3a3f650681f5bafe)). But it will run into this error:
```
[171/267] Generating sof-mtl-rt1019-rt5682-ctc.tplg
ALSA lib pre-process-class.c:167:(tplg_class_get_unique_attribute_name) No unique attribute in class 'mixout-gain-ctc-dai-copier-playback'

ALSA lib pre-process-object.c:1844:(tplg_build_object) error setting unique attribute value for 'mixout-gain-ctc-dai-copier-playback.0'

ALSA lib pre-process-object.c:2083:(tplg_pre_process_objects) Error building object Pipeline.mixout-gain-ctc-dai-copier-playback.0

Unable to pre-process configuration
```

Not sure did I miss anything?

Reply:
@eddy1021 I just compiled your PR and it worked fine. Have you upgraded your alsa-utils to the latest upstream version? Alternatively, I think our CI docker image is also updated to be able to compile with the latest alsa-utils.

Reply:
I updated `alsa-utils` and it's compiled now. Thanks!